runners/console/index.ts

```diff
@@ -60,6 +60,20 @@ export class Console extends Runner {
         return result;
     }
 
+    runCreateProject(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
```

Comment:
there is no differentiation on command.parameters[0].
I think we should throw an exception somehow to indicate if a template was used, which does not exist for easier debugging and understanding of failures